### PR TITLE
Update database-integration.md

### PR DESCRIPTION
**Pull request by selimsezr**
add DuckDB integration

### CODE REVIEW

File: en/guide/database-integration.md [hunk 1]
```diff
@@ -494,3 +495,32 @@ client.search({
   console.trace(error.message)
 })
 ```
+
+## DuckDB
+**Module**: [duckdb]([https://github.com/elastic/elasticsearch-js](https://github.com/duckdb/duckdb))
```
Comment (crandmck) [inline en/guide/database-integration.md:500]: This is a malformed md link.  There are 2 URLs here--pick one.
Why would Elastic be relevant?

**Review by crandmck — CHANGES_REQUESTED** [review]

**Comment by crandmck** [comment]
@jonchurch I don't see any reason not to add DuckDB.  WDYT?

Separately--and in the spirit of reducing/eliminating links or endorsements of 3rd party packages or products--I wonder if we should either move this page to the "Advanced" menu or just remove it entirely?  Clearly, it has useful info, but...?

**Comment by selimsezr** [comment]
It might be good to move it to the “Advanced” page. I tried for about 4 days to make the DuckDB connection with Nextjs and it didn't work. I did it much faster with Express. So I think it's important to write 3rd party applications. When there is an update or error, a PR/Issue can be created immediately.

I couldn't understand where the error was either, I only made changes in the .md file.

**Comment by crandmck** [comment]
@selimsezr If you would address my comment about the link, we can consider merging this.

**Comment by crandmck** [comment]
This hasn't been update for over a month, so closing.

**Pull request closed without merging**